src/test/java/com/softserveinc/ita/SearchTest.java

```diff
@@ -0,0 +1,37 @@
+package com.softserveinc.ita;
+
+import com.softserveinc.ita.pageobjects.HomePage;
+import com.softserveinc.ita.pageobjects.SearchResultPage;
+import com.softserveinc.ita.utils.runners.TestRunner;
+import org.testng.annotations.Test;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+public class SearchTest extends TestRunner {
+
+    @Test
+    public void verifySearchResultsTest() {
+        var homePage = new HomePage();
+        var requiredItem = "Xiaomi Redmi Note 10";
+        homePage
+                .closeAdvertisingBannerIfDisplayed()
+                .setTextInSearchBar(requiredItem)
+                .performSearch();
```

Comment:
is it possible, that performSearch method in the base page will always open the search results page instead of generic?

Reply:
i can make void or this way. If you are talking about it
public BasePage performSearch
click
send

return this;

Reply:
i can make thru void
public void performSearch() {
       $x("//button[contains(@class, 'search-form__submit')]").click();
    }

Reply:
or click on this method in test

Reply:
or return SearchResultPage?

Reply:
last the best